api/v1alpha1/openapi.yaml

```diff
@@ -949,6 +949,62 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Status'
+  /api/v1/devicelabels:
+    get:
+      tags:
+        - device-labels
+      description: |
+        Retrieves a distinct list of device labels.
+      operationId: listDeviceLabels
+      parameters:
+        - name: labelSelector
+          in: query
+          description: A selector to restrict the list of returned objects by their labels. Defaults to everything.
+          schema:
+            type: string
+        - name: fieldSelector
+          in: query
+          description: A selector to restrict the list of returned objects by their fields, supporting operators like '=', '==', and '!=' (e.g., "key1=value1,key2!=value2").
+          schema:
+            type: string
+        - name: limit
+          in: query
+          description: The maximum number of distinct labels to return in the response.
+          required: false
```

Comment:
Does not setting a limit have a bigger impact on service performance?  If so, we can make this field required with a maximum of 20, for example.